### PR TITLE
IJPS-64: new Database table - court_venue

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/IJPS-64

### Change description ###
A new Database table - court_venue

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @Autowired
+    private CourtVenueRepository courtVenueRepository;
+
+    @BeforeEach
+    void setUp() {
+        courtVenueRepository.deleteAll();
+    }
+
+    @Test
+    void shouldSaveSittingRecord() {
```
Comment (RebeccaBaker) [inline src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java:29]:
```suggestion
    void shouldSaveCourtVenue() {
```

File: src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    private CourtVenue createCourtVenue() {
+        return CourtVenue.builder()
+            .epimmsId("1234")
+            .hmctsServiceId("test")
+            .costCenterCode("456")
+            .build();
+    }
+
+    @Test
+    void shouldUpdateSittingRecordWhenRecordIsPresent() {
```
Comment (RebeccaBaker) [inline src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java:48]:
```suggestion
    void shouldUpdateCourtVenueWhenRecordIsPresent() {
```

**Review by RebeccaBaker — APPROVED** [review]
Minor suggestion for test names other than that looks good

File: src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java (hunk 1 above)
Comment (SomeshRA) [reply src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java:29]: Updated to shouldSaveCourtVenue

File: src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java (hunk 2 above)
Comment (SomeshRA) [reply src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java:48]: Updated to shouldUpdateCourtVenueWhenRecordIsPresent

**Pull request merged**